### PR TITLE
feat: add fixtures for tests

### PR DESCRIPTION
**Pull request by xXenvy**

### CODE REVIEW

File: src/tests/fixtures.zig [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+pub fn init() !Fixtures {
+    const allocator: std.mem.Allocator = std.testing.allocator;
+    const tracing_allocator: TracingAllocator = TracingAllocator.init(allocator);
+
+    const logger: Logger = try Logger.init(allocator, null, false);
+    const config: Config = try Config.load(allocator, null, null);
+    const persistance: PersistanceHandler = try PersistanceHandler.init(allocator, config, logger, null);
+
+    return Fixtures{ .allocator = allocator, .tracing_allocator = tracing_allocator, .logger = logger, .config = config, .persistance = persistance };
```
Comment (sectasy0) [inline src/tests/fixtures.zig:26]: Please, format this line by adding `,` after the last field.

File: src/tests/server/persistance.zig [hunk 2]
```diff
@@ -1,33 +1,27 @@
...
     std.fs.cwd().deleteFile("./tmp/persist/dump_123.zcpf") catch {};
 
-    var config = try Config.load(std.testing.allocator, null, null);
-    defer config.deinit();
-
-    const logger = try Logger.init(std.testing.allocator, null, false);
+    var fixtures = try Fixtures.init();
+    defer fixtures.deinit();
 
     var persister = try PersistanceHandler.init(
```
[the remark this answers is not in the document]
Comment (xXenvy) [reply src/tests/server/persistance.zig]: already done.

File: src/tests/fixtures.zig [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+pub fn create_memory_storage(self: *Fixtures) MemoryStorage {
```
[the remark this answers is not in the document]
Comment (xXenvy) [reply src/tests/fixtures.zig]: Done in [600ccd4.](https://github.com/sectasy0/zcached/pull/8/commits/600ccd45dce6b70fa905d5e7e18855827477dca5)

File: src/tests/fixtures.zig (hunk 1 above)
Comment (xXenvy) [reply src/tests/fixtures.zig]: Done in [600ccd4.](https://github.com/sectasy0/zcached/pull/8/commits/600ccd45dce6b70fa905d5e7e18855827477dca5)

**Pull request merged**